### PR TITLE
BTA-2054 | Play 2.6 upgrade

### PR DESCRIPTION
**Pull request by suyangyu**

### CODE REVIEW

File: app/config/FrontendAppConfig.scala [hunk 1]
```diff
@@ -22,83 +22,95 @@ import com.google.inject.{Inject, Singleton}
...
 
 @Singleton
-class FrontendAppConfig @Inject() (override val runModeConfiguration: Configuration, environment: Environment) extends ServicesConfig with PortalUrlBuilder {
+class FrontendAppConfig @Inject()(val runModeConfiguration: Configuration,
+                                  environment: Environment,
+                                  servicesConfig: ServicesConfig,
+                                  override val languageUtils: LanguageUtils) extends PortalUrlBuilder {
 
-  override protected def mode = environment.mode
+  import servicesConfig._
```
Comment (liampan) [inline app/config/FrontendAppConfig.scala]: can this live with the other imports?

File: app/utils/PortalUrlBuilder.scala [hunk 2]
```diff
@@ -17,20 +17,28 @@
...
   private def appendLanguage(url: String)(implicit request: Request[_]) = {
-    val lang = if (LanguageUtils.getCurrentLang == LanguageUtils.Welsh) "lang=cym" else "lang=eng"
+    val lang = if (languageUtils.getCurrentLang == WELSH) "lang=cym" else "lang=eng"
     val token = if (url.contains("?")) "&" else "?"
     s"$url$token$lang"
   }
 }
+
+object PortalUrlBuilder {
+  val WELSH: Lang = Lang("cy")
```
Comment (liampan) [inline app/utils/PortalUrlBuilder.scala]: why all caps?

File: app/config/FrontendAppConfig.scala (hunk 1 above)
Comment (suyangyu) [reply app/config/FrontendAppConfig.scala]: no, cos it's importing the methods from within the servicesConfig instance

File: app/utils/PortalUrlBuilder.scala (hunk 2 above)
Comment (suyangyu) [reply app/utils/PortalUrlBuilder.scala]: To denote that it's a static constant

**Pull request merged**